ikarus/controlroutines/adaptivestepsizing.hh

```diff
@@ -0,0 +1,40 @@
+// SPDX-FileCopyrightText: 2021-2023 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#pragma once
+
+#include <ikarus/controlroutines/pathfollowingfunctions.hh>
+#include <ikarus/solver/nonlinearsolver/solverinfos.hh>
+
+namespace Ikarus::AdaptiveStepSizing {
+  template <typename ScalarType = double>
```

Comment:
I thing templating both of these functions by some scalartype is quite useless, at least right now

Reply:
Yeah.. I have removed it